gluten-ut/spark32/src/test/scala/org/apache/spark/sql/execution/GlutenSQLWindowFunctionSuite.scala

```diff
@@ -16,6 +16,71 @@
  */
 package org.apache.spark.sql.execution
 
+import io.glutenproject.execution.WindowExecTransformer
+
 import org.apache.spark.sql.GlutenSQLTestsTrait
+import org.apache.spark.sql.Row
+
+class GlutenSQLWindowFunctionSuite extends SQLWindowFunctionSuite with GlutenSQLTestsTrait {
+
+  import testImplicits._
+
+  val customerData: Seq[(Int, Int)] = Seq[(Int, Int)](
+    (4553, 11),
+    (4953, 10),
+    (35403, 5),
+    (35803, 12),
+    (60865, 5),
+    (61065, 13),
+    (127412, 13),
+    (148303, 10),
+    (9954, 5),
+    (95337, 12)
+  )
 
-class GlutenSQLWindowFunctionSuite extends SQLWindowFunctionSuite with GlutenSQLTestsTrait {}
+  test("Literal in window partition by and sort") {
```

Comment:
> in the current PR, only partitionExprs and sortExprs are considered

Is it possible to also add test for window expressions? At least to make sure it is runnable with fallback.